app/src/Entity/Game.php

```diff
@@ -35,8 +42,9 @@ class Game {
    * @ORM\Column(type="boolean")
    */
   private $isVictory;
-  
+
   public function __construct($resolvedGrid, $currentGrid, $isVictory=false) {
+    $this->token = bin2hex(random_bytes(self::TOKEN_LENGTH));
```

Comment:
Domain objects needs to be "neutral". They must not contains any "logic". I'll create this token in the "new" method of the API where the object is instancied for the first time and give him the value with "setToken".